mediorum/server/repair.go

```diff
@@ -321,7 +321,7 @@ func (ss *MediorumServer) repairCid(cid string, placementHosts []string, tracker
 	// don't delete if we replicated the blob within the past week
 	wasReplicatedThisWeek := attrs.CreateTime.After(time.Now().Add(-24 * 7 * time.Hour))
 
-	if !isPlaced && !ss.Config.StoreAll && tracker.CleanupMode && alreadyHave && myRank > ss.Config.ReplicationFactor+2 && !wasReplicatedThisWeek {
+	if !isPlaced && !ss.Config.StoreAll && tracker.CleanupMode && alreadyHave && myRank > ss.Config.ReplicationFactor*3 && !wasReplicatedThisWeek {
```

Comment:
*3 again? are we worried this is too much commitment?

Reply:
Added a commit to make the threshold dynamic based on free disk space.  So nodes with larger disks can accommodate files that can't be placed on smaller nodes.  Just guessed at numbers for now based on looking at healthz.